python/electrostatic_force.py

```diff
@@ -0,0 +1,362 @@
+from tkinter import *
+import compute
+def com1():
+    global lavel
+    charge1val=Charge1.get()
+    #print(charge1val)
+    charge2val=Charge2.get()
+    #print(charge2val)
+    root.destroy()
+    root1=Tk()
+    global flag
+    root1.title("OUTPUT")
+    root1.geometry("7000x6000")
+    root1.minsize(500,600)
+    canvas=Canvas(root1,width=2000,height=2200,bg="sky blue")
+    canvas.place(x=0,y=0)
+    M=StringVar()
+    x_1,y_1,x_2,y_2=550,200,650,300  #coordinates for the big circle
+    a_1,b_1,a_2,b_2=730,220,790,280  #coordinates for the small circle
+    global countsmallclick,countbigclick
+    #global bigclick,smallclick
+    bigclick=0
+    smallclick=0
```

Comment:
Codacy found an issue: [local variable 'smallclick' is assigned to but never used (F841)](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279653)